packages/app/src/components/DownloadMenu/PlainDataSourceSubMenu.js

```diff
@@ -0,0 +1,57 @@
+import i18n from '@dhis2/d2-i18n'
+import { MenuItem, MenuSectionHeader } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import {
+    DOWNLOAD_TYPE_PLAIN,
+    ID_SCHEME_UID,
+    ID_SCHEME_CODE,
+    ID_SCHEME_NAME,
+} from './constants.js'
+
+export const PlainDataSourceSubMenu = ({
+    download,
+    format,
+    label,
+    ...menuItemProps
```

Comment:
`menuItemProps` is unused and should be removed?

Reply:
🤔 it's passed to `<MenuItem>`.

Reply:
What I mean is that it's never set in the first place (look at where this component is being used)

Reply:
Just to confirm. I agree with @martinkrulltott that these `menuItemProps` can and should be removed. They are currently unused, and it would probably make more sense to add explicit props if we do need to pass anything to the menu item later on.